examples/webgl_materials_envmaps_hdr.html

```diff
@@ -275,6 +278,23 @@
 
 					}
 
+
+					var side = standardMaterial.side;
+
+					switch ( params.side ) {
+
+						case 'front': side = THREE.FrontSide; break;
+						case 'back': side = THREE.BackSide; break;
+						case 'double': side = THREE.DoubleSide; break;
+	
+					}
+
+					if( standardMaterial.side !== side ) {
+						
+						standardMaterial.side = side;
+						standardMaterial.needsUpdate = true;
```

Comment:
Oh, yes. It's the `defines` that are the culprit. I think we implemented it that way to avoid an if-statement in the shader.